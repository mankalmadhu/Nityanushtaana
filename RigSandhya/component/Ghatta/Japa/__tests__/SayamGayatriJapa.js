import React from 'react';
import ಸಾಯ೦ಗಾಯತ್ರೀ from '../SayamGayatriJapa';
import Renderer from 'react-test-renderer';

jest.mock('../../Ghatta.vinyasa');
jest.mock('../../../Vishaya/Vishaya', () => 'ವಿಷಯ');
jest.mock('../../Gayatri/Trihruti', () => 'ತ್ರಿಹೃತಿ');
jest.mock('../../Sankalpa/SayamJapaSankalpa', () => 'ಸಾಯ೦ಜಪಸ೦ಕಲ್ಪ');
jest.mock('../Aavahana', () => 'ಆವಾಹನ');
jest.mock('../../Gayatri/GayatriMantra', () => 'ಗಾಯತ್ರೀಮ೦ತ್ರ');
jest.mock('../Karanyaasa', () => 'ಕರನ್ಯಾಸ');
jest.mock('../Anganyaasa', () => 'ಅ೦ಗನ್ಯಾಸ');
jest.mock('../Digbhanda', () => 'ದಿಗ್ಬ೦ಧನ');
jest.mock('../SayamSwaroopa', () => 'ಸಾಯ೦ಸ್ವರೂಪ');
jest.mock('../Mantrartha', () => 'ಮ೦ತ್ರಾರ್ಥ');
jest.mock('../Digvimoka', () => 'ದಿಗ್ವಿಮೋಕ');
jest.mock('../Muktaya', () => 'ಮುಕ್ತಾಯ');
 
test("ಸಾಯ೦ಗಾಯತ್ರೀ", () => {
  expect(Renderer.create(
    <ಸಾಯ೦ಗಾಯತ್ರೀ />
  ).toJSON()).toMatchSnapshot();

}); 