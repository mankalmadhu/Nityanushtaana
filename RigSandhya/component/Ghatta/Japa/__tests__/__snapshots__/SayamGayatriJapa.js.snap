// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಸಾಯ೦ಗಾಯತ್ರೀ 1`] = `
<View>
  <ಶೀರ್ಷಿಕೆ>
    ಗಾಯತ್ರೀ ಜಪ
  </ಶೀರ್ಷಿಕೆ>
  <ತ್ರಿಹೃತಿ />
  <ತ್ರಿಹೃತಿ />
  <ಆವಾಹನ />
  <ಸಾಯ೦ಜಪಸ೦ಕಲ್ಪ />
  <ಕರನ್ಯಾಸ />
  <ಅ೦ಗನ್ಯಾಸ />
  <ದಿಗ್ಬ೦ಧನ />
  <ಸಾಯ೦ಸ್ವರೂಪ />
  <ಮ೦ತ್ರಾರ್ಥ />
  <ಜಪ />
  <ಕರನ್ಯಾಸ />
  <ಅ೦ಗನ್ಯಾಸ />
  <ದಿಗ್ವಿಮೋಕ />
  <ಮುಕ್ತಾಯ />
</View>
`;
